<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="DESKTOP-VSJMPEM" failures="1" tests="1" name="Default test(failed)" time="25.586" errors="0" timestamp="2024-02-17T17:14:36 IST">
  <testcase classname="com.testcases.LoginPageTest" name="loginTest" time="3.764">
    <failure type="java.lang.AssertionError" message="expected [https://www.saucedemo.com/inventory.html] but found [https://www.saucedemo.com/]">
      <![CDATA[java.lang.AssertionError: expected [https://www.saucedemo.com/inventory.html] but found [https://www.saucedemo.com/]
at com.testcases.LoginPageTest.loginTest(LoginPageTest.java:43)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 31 stack frames]]>
    </failure>
  </testcase> <!-- loginTest -->
</testsuite> <!-- Default test(failed) -->
